releases/fuji/compose-files/docker-compose-fuji-no-secty.yml

```diff
@@ -49,9 +49,6 @@ services:
 
   consul:
     image: consul:1.3.1
-    ports:
-      - "8400:8400"
-      - "8500:8500"
```

Comment:
Should we retro-change the Fuji version? Maybe the decision should be called for attention by the groups?

Reply:
If we did this it would have to be a point release, which I don't think it warrants as of security hardening is evolving release to release, IMHO. @jpwhitemn thoughts?

Reply:
@jim-wang-intel I wasn't sure if this would be considered a "security fix" and therefore worth backporting to the current release (though current release is Geneva isn't it 😬  ) as well as nightly